playbooks/ansible-test-sanity-base/files/bootstrap-asa.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
we shouldn't need to bootstrap any appliances, as sanity will not need them. Lets remove theses